opm/simulators/linalg/ISTLSolverEbos.hpp

```diff
@@ -645,29 +646,112 @@ class WellModelMatrixAdapter : public Dune::AssembledLinearOperator<M,X,Y>
         }
 
         /// Zero out off-diagonal blocks on rows corresponding to overlap cells
-        /// Diagonal blocks on ovelap rows are set to diag(1e100).
+        /// Diagonal blocks on ovelap rows are set to diag(1.0).
```

Comment:
It seems like this function is not called anymore. Shouldn't we delete it?

Reply:
Yes I will delete it. There is a copy of this function in ISTLSolverEbosFlexible.hpp that should stay.